conans/client/cache/remote_registry.py

```diff
@@ -154,12 +175,27 @@ def _get_by_url(self, url):
 
     def rename(self, remote_name, new_remote_name):
         if new_remote_name in self._remotes:
-            raise ConanException("Remote '%s' already exists" % new_remote_name)
+            raise ConanException("Remote '%s' already exists" %
+                                 new_remote_name)
 
         remote = self._remotes[remote_name]
-        new_remote = Remote(new_remote_name, remote.url, remote.verify_ssl)
-        self._remotes = OrderedDict([(new_remote_name, new_remote) if k == remote_name
-                                     else (k, v) for k, v in self._remotes.items()])
+        new_remote = Remote(new_remote_name, remote.url, remote.verify_ssl,
+                            remote.disabled)
+        self._remotes = OrderedDict([
+            (new_remote_name, new_remote) if k == remote_name else (k, v)
+            for k, v in self._remotes.items()
+        ])
```

Comment:
Try to avoid this format changes in a feature PR, let's keep the changes related to functionality isolated from those related to formatting (it is fine to submit a PR just with format changes if needed: was the line too long?) (same for the `raise` above)